Entities/Scene.cpp

```diff
@@ -3017,17 +3053,17 @@ void Scene::Update()
 		}
 	}
 
-    // Do full update every two minutes
-    if (m_FullPathUpdateTimer.IsPastSimMS(120000))
+    // Do a partial update every 10 seconds
```

Comment:
I think this comment is pretty unnecessary, the code is clear enough without it.

Reply:
I still want a comment here, but a better one